cfme/v2v/migration_plans.py

```diff
@@ -468,6 +468,11 @@ def wait_for_state(self, state):
         except TimedOutError:
             logger.info("Wait for state :%s timed out", state)
 
+    def delete_completed_plan(self):
```

Comment:
Thought for the future, doesn't have to be included in this PR. Would it be possible to merged `delete_completed_plan` and `delete_not_started_plan` into a single `delete` method? 

As a user, I would expect that `delete` will delete the plan regardless of its status. Something like
```python
def delete(self):
    if self.completed():
        self.delete_completed_plan()
    else:
        self.delete_not_started_plan()
```

Reply:
@john-dupuy : Yeah I thought it too but the list to delete plan is different in each page so it need some time so created this method for time being . will do that in next PR :+1: